app/policies/submission_policy.rb

```diff
@@ -27,4 +28,12 @@ def not_a_student?
   def before_release?
     !record.current_result.released_to_students
   end
+
+  def collect?
+    allowed_to?(:manage_submissions?, with: GraderPermissionPolicy)
+  end
+
+  def allowed_to_run_tests?
```

Comment:
We shouldn't use "allowed_to" in policy names. (Since we call `allowed_to` on the policy)

Rename this to `run_tests?`

Reply:
We already have run_tests? in the same policy file. So I renamed this into can_run_tests?